cpp/ql/lib/experimental/semmle/code/cpp/semantic/analysis/RangeAnalysisStage.qll

```diff
@@ -157,7 +186,7 @@ module RangeStage<RangeSig RangeParam> {
      */
     cached
     predicate possibleReason(SemGuard guard) {
-      guard = boundFlowCond(_, _, _, _, _) or guard = semEqFlowCond(_, _, _, _, _)
+      guard = boundFlowCond(_, _, _, _, _) or guard = UtilParam::semEqFlowCond(_, _, _, _, _)
```

Comment:
Could we privately import `UtilParam` inside the `RangeStage` module to avoid prefixing with `UtilParam`? (And similarly for the other module parameters.)

Reply:
What was the conclusion here? Is this possible, or not?

Reply:
It's done now - see [this commit](https://github.com/github/codeql/pull/11728/commits/359b4282bae0c3652e65fb1f0ec32dd39506ea71)

Reply:
Not in ModulusAnalysis, where I'd also expect it.

Reply:
Looking at this again. I'm somewhat split on this. On the one hand, having the imports makes the code more compact. On the other hand, without the imports it's clearer where things are coming from. So I'm fine with leaving things as-is in the modulus analysis.